metaci/build/models.py

```diff
@@ -704,16 +704,23 @@ def _get_flow_options(self) -> dict:
         if (
             self.build.plan.role == "push_sandbox" and self.build.release
         ):  # override lives in MetaCI
-            options["push_sandbox"] = {
-                "version": f"{self.build.release.version_number}",
-            }
+            try:
+                options["push_sandbox"] = {
+                    "version": f"{self.build.release.version_number}",
+                    "start_time": self.build.release.implementation_steps.get(plan__id=self.build.plan.id).push_time.strftime("%Y-%m-%dT%H:%M")
+                }
+            except Exception:
+                raise Exception(f"Ensure that you have declared an implementation step with the {self.build.plan.role}.")
```

Comment:
Since we're filtering by plan id, we should probably mention the expected plan name rather than a role?